DemoApp/routes/report.router.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const router = express.Router();
+const multer = require("multer");
+const uploadFile = require("../middlewares/upload");
```

Comment:
Change configuration of upload.sing() to upload.any() then you will be able to upload multiple files with no parameter restrictions.